src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Square extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition.